README.rst

```diff
@@ -346,25 +356,21 @@ Configuration Files
 As with any CLI where the configuration can get rather complex, you might want to use a configuration
 file to define some or all of your desired behavior.
 
-.. warning::
-
-   There is a `known issue with config file parsing in Windows`_. Including single or double quote
-   characters in a config file on Windows will fail until we fix this issue. Please let us know
-   if this impacts you in the linked GitHub issue.
+Configuration files are supported by using Python's native `argparse file support`_, which allows
+you to write configuration files exactly as you would enter arguments in the shell. 
 
-Configuration files are supported using Python's native `argparse file support`_, which allows
-you to write configuration files exactly as you would enter arguments in the shell. Configuration
-file references passed to ``aws-encryption-cli`` are identified by the ``@`` prefix and the
-contents are expanded as if you had included them in line. Configuration files can have any
+To pass configuration file references to the ``aws-encryption-cli``, prefix the file name with ``@``. The
+configuration file contents are expanded as if you had included them in line. Configuration files can have any
 name you desire.
 
 .. note::
 
-   In PowerShell, you will need to escape the ``@`` symbol so that it is sent to ``aws-encryption-cli``
-   rather than interpreted by PowerShell.
+   In PowerShell, you need to escape the ``@`` symbol (`@) so that the configuration file reference 
```

Comment:
The `@` in parentheses needs to be enclosed in double backticks. A single backtick will actually be interpreted by Sphinx as incomplete syntax and be ignored. Double backticks indicate pre-formatted text or in-line code sample, and a single backtick can be included inside them as well.